api/README.md

```diff
@@ -613,3 +614,89 @@ __Example JSON__
 	]
 }
 ```
+
+## Querying test results
+
+### /api/search
+
+Search for test results over some set of test runs.
+
+_This API is currently in staging, not production._
+
+__Parameters__
+
+__`run_ids`__ : Array-separated list of numerical ids associated with the runs
+over which to search. IDs associated with runs can be obtained by querying the
+`/api/runs` API. Defaults to the default runs returned by `/api/runs`.
+
+__`q`__: Query string for search. Only results data for tests that contain the
+`q` value as a substring of the test name will be returned. Defaults to the
+empty string, which will yield all test results for the selected runs.
+
+#### Examples
+
+- https://staging.wpt.fyi/api/search?run_ids=6311104602963968,5132783244541952&q=xyz
+
+__Example JSON__
+
+<details>
```

Comment:
@jugglinmike what am I doing wrong? ( see https://github.com/web-platform-tests/wpt.fyi/blob/748ee3c1e134d69565abc8f0bda661a66feccedc/api/README.md#apisearch )

Reply:
May be a HTML<->GFM parsing thing. Try separating the tags and the backticks with newlines. Also, don't forget a `<summary>`.

Reply:
Thanks! More newlines seems to fix it.

Reply:
Newlines are the developer's club soda